### ISSUE TITLE
[Vue warn]: Invalid key "sort" in validation rules object for prop "options".

### ISSUE THREAD

**Issue by jhalarde** [David-Desmaisons/Vue.Isotope#44]
component is squawking about invalid key sort. Removing the sort variable in option props object seems to remove the warning. please fix. I'm using vue v2.5.11

**Comment by David-Desmaisons** [comment]
jsfiddle?

**Comment by gborcherds** [comment]
I'm getting this too on the latest version.

**Comment by David-Desmaisons** [comment]
If you guys can create a fidle it will help

**Comment by gborcherds** [comment]
I think it's your prop validation.

```javascript

options: {
        type: Object,
        default: {
          layoutMode: 'masonry',
          masonry: {
            gutter: 10
          }
        },
        sort: String
      },

```

The sort looks like it's in the wrong place. It should either be after the next bracket because it's its own prop, or it should be under gutter if it's part of the options prop. I think it can probably be removed completely though because you aren't using it as a prop, you have a method named "sort," but I couldn't figure out what if any use for the prop sort was used for.

**Comment by David-Desmaisons** [comment]
@gborcherds thanks for the explanation and details, you are probably correct, this props is not needed.

**Comment by David-Desmaisons** [comment]
Fixed in version 3.1.1

**Comment by gborcherds** [comment]
Thanks @David-Desmaisons. I appreciate it.
